lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -194,7 +197,7 @@ def test_todays_date_block(self):
         'info',
         'openedx.course_experience.course_home',
     )
-    @override_flag(UNIFIED_COURSE_EXPERIENCE_FLAG, active=True)
+    @override_flag_plus(course_experience_config(), UNIFIED_COURSE_EXPERIENCE_FLAG, active=True)
```

Comment:
I'm now leaning towards calling our new concept feature flags, and this would then be:

    @override_feature_flag(COURSE_EXPERIENCE_FEATURE_FLAG, active=True)